CONTRIBUTING.md

```diff
@@ -2,12 +2,8 @@
 
 ## Adding a feature or bugfix to Valtimo
 
-1. Discuss the (approach for the) feature or bugfix with someone from the Valtimo team. This is to reduce the work that
-   is required, as it can both reduce the chances of feedback and allows for the Valtimo team to provide some pointers
-   on how to proceed. It is not necessary to do this for smaller bugfixes.
-2. Implement the feature/bugfix and create a PR to the development branch. Add all 
-   [team Valtimo developers](https://github.com/orgs/valtimo-platform/teams/valtimo-product-team/members) to this PR
-   as reviewers.
+1. Create an issue in the [Valtimo issues repository](https://github.com/valtimo-platform/valtimo-issues) to discuss your contribution. This is to reduce the work that is required, as it can both reduce the chances of feedback and allows for the Valtimo team to provide some pointers on how to proceed.
```

Comment:
Not so much something that has to be fixed as per this PR, but should definitely be added to the new repo: Guidelines on how to report bugs, requesting a feature and how to announce a PR.